src/styles/blocks/page.css

```diff
@@ -0,0 +1,9 @@
+.page {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+.page__search {
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). It's a good practice to stick to either top or bottom margins for elements to avoid potential margin collapse issues.